src/local_scheduler/local_scheduler.cc

```diff
@@ -78,28 +77,25 @@ int force_kill_worker(event_loop *loop, timer_id id, void *context) {
  *        to clean up its own state.
  * @return Void.
  */
-void kill_worker(LocalSchedulerClient *worker, bool cleanup) {
+void kill_worker(LocalSchedulerState *state,
```

Comment:
I don't think we need to `state` as an argument here (`LocalSchedulerClient`s should have a reference to it).